framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/model/components/Build.java

```diff
@@ -86,4 +91,21 @@ public String getExtensionImageInstallDir() {
     public void setExtensionImageInstallDir(String extensionImageInstallDir) {
         this.extensionImageInstallDir = extensionImageInstallDir;
     }
+
+    public void addWebhooks(Webhooks webhooks) {
```

Comment:
Yes, they are needed to correctly set up KieApp used in tests. They were some changes for KieApp that caused test failures because not correctly generated KieApp

Reply:
Do you mean that this is used internally when marshalling/unmarshalling the Build instance?